app/src/main/java/agersant/polaris/PolarisPlaybackService.java

```diff
@@ -53,10 +56,43 @@
 	private Handler autoSaveHandler;
 	private Runnable autoSaveRunnable;
 
+	private Runnable dataUpdateRunnable;
+	private Handler dataUpdateHandler;
+
 	private API api;
 	private PolarisPlayer player;
 	private PlaybackQueue playbackQueue;
 
+	private MediaSessionCompat mediaSession;
+
+	private class Callback extends MediaSessionCompat.Callback {
```

Comment:
Similar request here, can you name this something more specific than Callback?

Reply:
It's now named MediaSessionCallback